console/src/telemetry/Actions.ts

```diff
@@ -159,10 +159,14 @@ const updateConsoleNotificationsState = (updatedState: NotificationsState) => {
     const restState = getState().telemetry.console_opts;
     const headers = dataHeaders(getState);
     let userType = 'admin';
-    if (headers?.[HASURA_COLLABORATOR_TOKEN]) {
-      const collabToken = headers[HASURA_COLLABORATOR_TOKEN];
+    const headerHasAdminToken = Object.keys(headers).find(
+      header => header.toLowerCase() === HASURA_COLLABORATOR_TOKEN
+    );
+    if (headerHasAdminToken) {
+      const collabToken = headers[headerHasAdminToken];
       userType = getUserType(collabToken);
     }
+
```

Comment:
```
if (userType !== 'admin') {
      const currentState = restState?.console_notifications;
      const currentState = restState?.console_notifications;
      if (Object.keys(currentState ?? {}).length > 1) {
      if (Object.keys(currentState ?? {}).length > 1) {
        composedUpdatedState = {
        composedUpdatedState = {
          ...restState,
          ...restState,
          console_notifications: {
          console_notifications: {
            ...currentState,
            ...currentState,
            [userType]: updatedState,
            [userType]: updatedState,
          },
          },
        };
        };
      }
      }
    }
    }
```

When an admin tries to click on the notification bar and an update to the database is triggered, the payload only has `admin` related information and other collaborators details are removed. I am sensing we should add the data of other collaborators when the current user is `admin`

Reply:
@karthikvt26 I think this case would be resolved, once I make the changes for the cases that you've mentioned here: https://github.com/hasura/graphql-engine/pull/6067#pullrequestreview-519327097